src/chttpd_db.erl

```diff
@@ -248,9 +264,21 @@ delete_db_req(#httpd{}=Req, DbName) ->
         throw(Error)
     end.
 
+get_db_options(DbName) ->
+    IsReplicatorDb = DbName == config:get("replicator", "db", "_replicator"),
+    IsUsersDb = DbName ==config:get("chttpd_auth", "authentication_db", "_users") orelse
+    binary_to_list(mem3:dbname(DbName)) == config:get("chttpd_auth", "authentication_db", "_users"),
```

Comment:
This definitely need to be simplified: no need to get the same config value twice. Also, without indention this a bit confuses.